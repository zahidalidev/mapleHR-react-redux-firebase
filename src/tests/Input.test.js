import { shallow } from 'enzyme'

import Input from 'components/input'

const valudInputComponent = <Input item='email' formik={{ errors: [], values: [] }} />

describe('Input', () => {
  test('render', () => {
    expect(shallow(valudInputComponent).exists()).toBe(true)
  })

  test('should render input component', () => {
    expect(shallow(valudInputComponent).getElements()).toMatchSnapshot()
  })
})
