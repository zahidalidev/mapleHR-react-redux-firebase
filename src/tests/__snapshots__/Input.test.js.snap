// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input should render input component 1`] = `
Array [
  <div
    className="input-wrapper"
  >
    <input
      autoComplete="email"
      className="input-field undefined"
      name="email"
      placeholder="email"
      type="email"
    />
  </div>,
]
`;
